sdk/ml/azure-ai-ml/azure/ai/ml/__init__.py

```diff
@@ -57,6 +60,9 @@
     "load_datastore",
     "load_model",
     "load_environment",
+    "load_featureset",
+    "load_feature_store",
```

Comment:
load_feature_store vs load_featurestore
Would be good to make this and load_featurestore_entity consistent

Reply:
the classes are like :
FeatureStore
Featureset
FeaturestoreEntity

Reply:
Could be updated later after discussion with feature store crew